src/LaraCart.php

```diff
@@ -557,12 +564,16 @@ public static function formatMoney($number, $locale = null, $internationalFormat
      *
      * @return string
      */
-    public function feeTotals($format = true)
+    public function feeTotals($format = true, $withTax = false)
```

Comment:
What if you want to calculate the total of tax cost (items + fees). Then you need this option. Am I right ?